src/components/badge/_badge.scss

```diff
@@ -0,0 +1,15 @@
+$base-height: 10px;
+
+.badge {
+  height: $base-height;
+  width: 15px;
+  background-color: white;
+  color: black;
+  border-radius: 1px;
+  font-size: 9px;
+  text-transform: uppercase;
+  margin-left: 3px;
+  line-height: $base-height !important;
```

Comment:
don't use !important. It is bad practise. If you need to override other css then add more level of selectors to increase css rank